src/test/java/uk/ac/ebi/eva/pipeline/io/readers/NonAnnotatedVariantsMongoReaderTest.java

```diff
@@ -124,4 +129,40 @@ private void assertDoesNotHaveVariantAnnotation(VariantWrapper variantWrapper)
         VariantAnnotation annotation = ((Variant) privateVariantField.get(variantWrapper)).getAnnotation();
         assertNull(annotation.getConsequenceTypes());
     }
+
+    @Test
+    public void shouldReadVariantsInAllStudies() throws Exception {
+        checkAllVariantsRead(EXPECTED_VARIANTS_IN_STUDY, STUDY_ID);
+    }
+
+    @Test
+    public void shouldReadVariantsInAllStudiesUsingNull() throws Exception {
+        checkAllVariantsRead(EXPECTED_VARIANTS_IN_DB, ALL_STUDIES);
```

Comment:
`ALL_STUDIES` is not null but an empty string. Maybe `shouldReadVariantsInAllStudiesWhenNoStudySpecified`?